tests/cmds/test_trustedactivities.py

```diff
@@ -0,0 +1,386 @@
+import json
+
+import pytest
+from py42.exceptions import Py42DescriptionLimitExceededError
+from py42.exceptions import Py42TrustedActivityConflictError
+from py42.exceptions import Py42TrustedActivityIdNotFound
+from py42.exceptions import Py42TrustedActivityInvalidCharacterError
+from py42.response import Py42Response
+
+from code42cli.main import cli
+
+TEST_RESOURCE_ID = 123
+ALL_TRUSTED_ACTIVITIES = """
+{
+    "trustResources": [
+        {
+            "description": "test description",
+            "resourceId": 456,
+            "type": "DOMAIN",
+            "updatedAt": "2021-09-22T15:46:35.088Z",
+            "updatedByUserUid": "user123",
+            "updatedByUsername": "username",
+            "value": "test"
+        }
+    ],
+    "totalCount": 10
+}
+"""
+
+TRUSTED_ACTIVITY_DETAILS = """
+{
+  "description": "test description",
+  "resourceId": 123,
+  "type": "DOMAIN",
+  "updatedAt": "2021-09-22T20:39:59.999Z",
+  "updatedByUserUid": "user123",
+  "updatedByUsername": "username",
+  "value": "test"
+}
+"""
+
+MISSING_ARGUMENT_ERROR = "Missing argument '{}'."
+MISSING_TYPE = MISSING_ARGUMENT_ERROR.format("[DOMAIN|SLACK]")
+MISSING_VALUE = MISSING_ARGUMENT_ERROR.format("VALUE")
+MISSING_RESOURCE_ID_ARG = MISSING_ARGUMENT_ERROR.format("RESOURCE_ID")
+RESOURCE_ID_NOT_FOUND_ERROR = "Resource ID '{}' not found."
+INVALID_CHARACTER_ERROR = "Invalid character in domain or slack workspace name"
+CONFLICT_ERROR = (
+    "Duplicate URL or workspace name, '{}' already exists on your trusted list."
+)
+DESCRIPTION_LIMIT_ERROR = "Description limit exceeded, max 250 characters allowed."
+
+
+@pytest.fixture
+def py42_response(mocker):
```

Comment:
`conftest` has the handy `create_mock_response()` to easily make fake responses.

Reply:
Ah, that makes sense, I'll use that instead.  I had grabbed this from the `cases` command lol

Reply:
yeah, I think Jules added that helper pretty recently, so the older cmds aren't using it.